airbnb_project/database/.github\workflows/deploy-image.yml

```diff
@@ -0,0 +1,42 @@
+name: Create and publish a Docker image
+
+on: push
+
+env:
+  REGISTRY: ghcr.io
```

Comment:
@gmartinsnull  I would like to discuss this topic with you whether we need the Github packages. I think we can answer this question if we have the decision made about how we going to deploy our app. Please feel free to advise me on this since I'm very new to Docker and CI/CD process.